docs/docs/guides/build/components/building-pipelines-with-components/testing-component-definitions.md

```diff
@@ -26,7 +26,7 @@ from pathlib import Path
 def my_project_component_defs(component_path) -> tuple[dg.Component, dg.Definitions]:
     # Project root is two parents up from the test file
     project_root = Path(__file__).parent.parent
-    tree = dg.ComponentTree.for_project(project_root=project_root)
+    tree = dg.ComponentTree.for_project(project_root)
```

Comment:
should we be using the like indirection to checked in python file that gets static analysis?